src/predefined-variables.ts

```diff
@@ -28,19 +58,19 @@ export function init ({gitData, argv}: PredefinedVariablesOpts): {[name: string]
         CI_COMMIT_REF_SLUG: gitData.commit.REF_NAME.replace(/[^a-z\d]+/ig, "-").replace(/^-/, "").slice(0, 63).replace(/-$/, "").toLowerCase(),
         CI_COMMIT_TIMESTAMP: gitData.commit.TIMESTAMP,
         CI_PIPELINE_CREATED_AT: new Date().toISOString().split(".")[0] + "Z",
-        CI_JOB_STARTED_AT: new Date().toISOString().split(".")[0] + "Z",
```

Comment:
This var is redundant since it'll be ovewritten by 
https://github.com/firecow/gitlab-ci-local/blob/47987f4843a33c4ddcbca0fbd17c793c8eba0903/src/job.ts#L462